package-extract/overlays/stage/imagestreamtag.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  name: package-extract
+spec:
+  tags:
+    - name: latest
+      from:
+        kind: DockerImage
+        name: "quay.io/thoth-station/package-extract:v0.6.0-dev"
```

Comment:
How will we manage versions? Will we run sed on all the templates to replace the version string?

Reply:
for now, its a manual and PR-based process to promote version from test to stage. in test the cyborgs will update as we release to quay.

Reply:
ack, thanks for the explanation